ape_solidity/compiler.py

```diff
@@ -52,6 +52,8 @@ def __init__(self):
 
 
 class SolidityCompiler(CompilerAPI):
+    import_map: Dict[str, str] = {}
```

Comment:
One thing to consider is how this works for dependencies

Reply:
Implemented a cache/reset for these remappings